docs/snapshotting/snapshot-editor.md

```diff
@@ -0,0 +1,61 @@
+# Snapshot editor
+
+The `snapshot-editor` is a program for modification of Firecracker snapshots.
+
+## Prior knowledge
+
+Firecracker snapshot consists of 2 files:
+
+- vmstate file: file with Firecracker internal data such as vcpu states,
+  devices states etc.
+- memory file: file with guest memory.
+
+`snapshot-editor` allows to modify each of these files separately.
```

Comment:
Right now it doesnt - we should write the documentation based on what currently is implemented (e.g. this tool deals with vmstate, and rebase-snap deals with memory file)

Reply:
I still we should just say "`snapshot-editor` (currently) allows to modify the `vmstate` file of aarch64 snapshots" instead of having multiple notes. Because this sentence here might be true in the future, but currently it is still false, so we should just merge it with the notes.

Reply:
Merged